### ISSUE TITLE
Accessibility - TextLanguageFieldset: rewrite with fieldset, hidden labels or area-labels

### ISSUE THREAD

**Issue by alina-lapina** [statisticsnorway/klass-subsets-client#190]
**Describe the bug**
For now the `Name`, `Description`, `VersionRationale` field is a set of inputs under a single label. In addition all the inputs generates with the same `id`. It is not accessibility compliant.

**App version**
Since v0.3.0, I guess, when ssb-component-library styles are applied.  

**To Reproduce**
1.  Åpen a subset with description in several languages in edit mode (form) or create one.
2.  Check the IDs to the textareas.
3.  Test accessibility flow.

**Expected behavior**
It should be clear for all the users where to type subset's name, description or vesion's rationale, where to change or choose language for each translation of the name.

**Screenshots**
![image](https://user-images.githubusercontent.com/21969896/103163687-8e082e00-4801-11eb-91e4-1aede20bf266.png)

**Suggested solution**
Solution 1: Implement it as a fieldset with a legend instead of label. The only problem is to remain the ssb's label styling.
Solution 2: Use area-label to make it accessible.
Solution 3: Put "hidden" label for each input (use style ".for_screen_readers")

**Additional context**
Doc ref (html5):
https://html.spec.whatwg.org/multipage/forms.html#the-label-element
The for attribute may be specified to indicate a form control with which the caption is to be associated. If the attribute is specified, the attribute's value must be the ID of a labelable element in the same tree as the label element. If the attribute is specified and there is an element in the tree whose ID is equal to the value of the for attribute, and the first such element in tree order is a labelable element, then that element is the label element's labeled control.

If the for attribute is not specified, but the label element has a labelable element descendant, then the first such descendant in tree order is the label element's labeled control.

The label element's exact default presentation and behavior, in particular what its activation behavior might be, if anything, should match the platform's label behavior. The activation behavior of a label element for events targeted at interactive content descendants of a label element, and any descendants of those interactive content descendants, must be to do nothing.
